CONTRIBUTING.md

```diff
@@ -60,6 +62,16 @@ The `rustup::currentprocess` module abstracts the global state that is
 `std::env::current_dir` and `std::process::exit` permitting threaded tests of
 the CLI logic; use `process()` rather than those APIs directly.
 
+### Clippy lints
+
+We don't enforce lint-free-ness in the gate, because it is a moving target, but
+we do ask that people run `cargo +nightly clippy --all-targets -- -D warnings`
+before submitting their code, to help keep things generally tidy.
```

Comment:
Overall para feels slightly ambiguous. What does "please run it but we don't enforce it" really mean?